doc/nrf/templates/cheat_sheet.rst

```diff
@@ -186,8 +186,11 @@ Read out an objects.inv file::
 Doxygen
 =======
 
-.. doxygenfunction:: profiler_init
-   :project: nrf
+We usually include doxygen groups::
```

Comment:
double ':' here ?

Reply:
Yes, to introduce code/verbatim in the following paragraph.

Reply:
ok